schemas/triniti/notify/mixin/browser-notification/latest.xml

```diff
@@ -20,6 +20,21 @@
         </description>
         <default>true</default>
       </field>
+      <field name="braze_campaign_id" type="uuid">
+        <description>
+          The Braze campaign identifier is required if you wish to track campaign stats.
+        </description>
+      </field>
+      <field name="braze_segment_id" type="uuid">
+        <description>
+          The Braze segment identifier that defines the target user segment for the message.
+        </description>
+      </field>
+      <field name="braze_message_variation_id" type="string">
+        <description>
+          The Braze message variation identifier is used when providing a campaign_id to specify which message variation this message should be tracked under (must be an Kindle/FireOS Push Message)
+        </description>
```

Comment:
@digitalartcrew this description is misleading. the bit about kindle/fireos. for each entry under [message objects](https://www.braze.com/docs/api/objects_filters/messaging/android_object) it lists that type as what it "must be"

Reply:
this is them saying, in a clunky way, that when you include a variation id with the `foo` config of a send, that variation must be a `foo` variation. ie if you provided the id of a ios variation with an android config, it would not work